playwright-ct.config.ts

```diff
@@ -0,0 +1,59 @@
+import { defineConfig, devices } from "@playwright/experimental-ct-react17";
```

Comment:
Is this location correct? I'm seeing this error in the file "Cannot find module '@playwright/experimental-ct-react17' or its corresponding type declarations."